<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureCreatorDialogBase</class>
 <widget class="QDialog" name="FeatureCreatorDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Terra Antiqua - Create Topo/Bathymetry</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="warningLabel">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="Tabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Parameters</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>479</width>
            <height>530</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Geographic feature type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="featureTypeBox"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Raster to be modified (Input):</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QgsMapLayerComboBox" name="baseTopoBox"/>
               </item>
               <item>
                <widget class="QToolButton" name="selectTopoBaseButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Layer with polygons for the geographic features:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QgsMapLayerComboBox" name="masksBox"/>
               </item>
               <item>
                <widget class="QToolButton" name="selectMasksButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="maxElevLabel">
               <property name="text">
                <string>Maximum sea depth (in m): </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="maxElevSpinBox">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>-9000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>0.000000000000000</double>
               </property>
               <property name="value">
                <double>-5750.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="minElevLabel">
               <property name="text">
                <string>Minimum sea depth  (in m):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="minElevSpinBox">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>-9000.000000000000000</double>
               </property>
               <property name="maximum">
                <double>0.000000000000000</double>
               </property>
               <property name="value">
                <double>-4000.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="shelfDepthLabel">
               <property name="text">
                <string>Maximum shelf depth (in m):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="shelfDepthSpinBox">
               <property name="minimum">
                <number>-1000</number>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
               <property name="value">
                <number>-200</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="shelfWidthLabel">
               <property name="text">
                <string>Shelf width (in km):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="shelfWidthSpinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>150</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="slopeWidthLabel">
               <property name="text">
                <string>Width of continental slope (in km):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="slopeWidthSpinBox">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="outputLabel">
             <property name="text">
              <string>Output file path:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QgsFileWidget" name="outputPath" native="true">
             <property name="fileWidgetButtonVisible" stdset="0">
              <bool>true</bool>
             </property>
             <property name="useLink" stdset="0">
              <bool>false</bool>
             </property>
             <property name="fullUrl" stdset="0">
              <bool>false</bool>
             </property>
             <property name="dialogTitle" stdset="0">
              <string/>
             </property>
             <property name="filter" stdset="0">
              <string/>
             </property>
             <property name="defaultRoot" stdset="0">
              <string>[Create temporary layer]</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="helpBox">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Here we shall see the help text for the tool.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="logText">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>runButton</tabstop>
  <tabstop>Tabs</tabstop>
  <tabstop>helpBox</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FeatureCreatorDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
